### PR TITLE
fix: DialogActions and SelectionFooter console warnings

### PR DESCRIPTION
**Pull request by gsamaniego41**

### CODE REVIEW

File: packages/components/src/DialogActions/DialogActions.tsx [hunk 1]
```diff
@@ -25,30 +22,22 @@ declare module '@mui/material/DialogActions/DialogActions' {
...
-  const { divider, ...others } = inProps
+export const DialogActions = React.forwardRef((inProps, ref) => {
+  const { className, divider, ...other } = inProps
   return (
     <MuiDialogActions
       ref={ref}
-      {...others}
-      sx={combineSxProps(getDialogActionsStyles(divider), others.sx)}
+      className={clsx(
+        divider === true && 'MonorailDialogActions-divider',
```
Comment (0x706b) [inline packages/components/src/DialogActions/DialogActions.tsx:31]: 🔧 Could you add this class as a value to a new `dialogActionsClasses` object?

```ts
// dialogActionsClasses.ts

import {DialogActionsClasses, dialogActionsClasses as muiDialogActionsClasses} from '@mui/material'

declare module '@mui/material/DialogActions/dialogActionsClasses' {
  export interface DialogActionsClasses {
    divider: string
  }
}

export const dialogActionsClasses: DialogActionsClasses = {
  ...muiDialogActionsClasses,
  divider: 'MonorailDialogActions-divider',
}
```

Comment (gsamaniego41) [reply packages/components/src/DialogActions/DialogActions.tsx:31]: Sure!

File: packages/components/src/DialogActions/dialogActionsClasses.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+declare module '@mui/material/DialogActions/dialogActionsClasses' {
+  export interface DialogActionsClasses {
+    divider: string
+  }
+}
+
+export const dialogActionsClasses: DialogActionsClasses = {
+  ...muiDialogActionsClasses,
+  divider: 'MonorailDialogActions-divider',
+}
```
Comment (0x706b) [inline packages/components/src/DialogActions/dialogActionsClasses.ts:13]: 🔧 Be sure to export this file too, in `components/src/DialogActions.ts`

Comment (gsamaniego41) [reply packages/components/src/DialogActions/dialogActionsClasses.ts:13]: Forgot about that one. Thanks for pointing it out!

**Review by 0x706b — APPROVED** [review]

**Pull request merged**